src/pages/HomePage.js

```diff
@@ -0,0 +1,12 @@
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+
+const HomePage = () => {
+  return (
+    <div>
+      <Header />
```

Comment:
you can remove Header and footer here if keep them in the App.js, applying them to App.js like you have will render them on all pages. 

by adding the code snippet to App.js that your teammate suggested should only render it on the pages you want to render.